src/opnsense/mvc/app/views/OPNsense/Firewall/alias.volt

```diff
@@ -633,10 +633,13 @@
                                                 </tr>
                                             </tbody>
                                         </table>
+                                        <div class="hidden" data-for="help_for_alias.frequency">
+                                                               <small>
+                                                {{lang._('The frequency that the list will be refreshed, in days + hours, so 1 day and 8 hours means the alias will be refreshed after 32 hours. ')}}
+                                            </small>
+                                        </div>
                                     </td>
-                                    <td>
-                                        <span class="help-block" id="help_block_alias.updatefreq"></span>
-                                    </td>
```

Comment:
^^ this looks like a mistake, these `<td/>` tags are used for feedback

Reply:
?? The first td is around the help click icon, same as alias name. The second block is around the inputs for day and hour with the hidden help within that block. Looks OK to me and works fine.

Reply:
well, it wouldn't if there where validations set to updatefreq, such as 
![image](https://user-images.githubusercontent.com/9783985/81931604-a9178c00-95ea-11ea-9f75-1709651d1c82.png)

in the name

Reply:
So why are they there around the alias.enabled help icon too?

Reply:
same reason, 3 column table, normally our grids are generated automatically (which makes sure the feedback column is always there) https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/views/layout_partials/form_input_tr.volt#L98-L100

trust me, it should be there....

Reply:
Well if I remove them it doesn't work. You're the expert show me how it should be done.